components/collapse/nz-collapse-panel.component.ts

```diff
@@ -44,12 +48,23 @@ import { NzCollapseComponent } from './nz-collapse.component';
   }
 })
 
-export class NzCollapsePanelComponent implements OnInit, OnDestroy {
+export class NzCollapsePanelComponent implements OnInit, AfterViewInit, OnDestroy {
   @Input() @InputBoolean() @HostBinding('class.ant-collapse-item-active') nzActive = false;
   @Input() @InputBoolean() @HostBinding('class.ant-collapse-item-disabled') nzDisabled = false;
   @Input() @InputBoolean() nzShowArrow = true;
   @Input() nzHeader: string | TemplateRef<void>;
   @Output() readonly nzActiveChange = new EventEmitter<boolean>();
+  @Input() set nzExpandedIcon(value: TemplateRef<void>) {
```

Comment:
Not necessary to validate whether `nzExpandedIcon` is a template ref. 

And you should also let users can pass a string and pass it to an `nz-icon`'s `type` prop for a simpler API.

You can also pass `nzActive` implicitly to the template, so users can have different icons for active and imactive panels.

Reply:
done